app/components/ak-modal/styles.scss

```diff
@@ -0,0 +1,17 @@
+.modal-header {
+  display: grid;
+  grid-template-columns: 1fr 30px;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
```

Comment:
use em or rem instead of px...its more portable across devices...eg from mac retina to normal screen

Reply:
The height of header is something like fixed, it may not be required to adopt based on screen size.